src/certificate.ts

```diff
@@ -109,18 +112,20 @@ function renderFirstPage(
     font: 'src/assets/fonts/OpenSans/OpenSans-SemiBold.ttf',
   });
 
-  text(doc, {
-    text: '(Web Development)',
-    x: 0,
-    y: 366,
-    options: {
-      width: A4SIZE[0],
-      align: 'center',
-    },
-    fontSize: 17,
-    fillColor: SECONDARY_TEXT_COLOR,
-    font: 'src/assets/fonts/OpenSans/OpenSans-Light.ttf',
-  });
+  if (course.type !== 'data') {
+    text(doc, {
+      text: '(Web Development)',
```

Comment:
Is it the same text for Java?

Reply:
No, this is mentioned in issue [#23](https://github.com/neuefische/certificates/issues/23) and will not be adressed in this PR